lib/mix/lib/mix/tasks/compile.app.ex

```diff
@@ -199,73 +208,122 @@ defmodule Mix.Tasks.Compile.App do
   end
 
   defp validate_properties!(properties) do
-    Enum.each properties, fn
+    Enum.each(properties, fn
       {:description, value} ->
         unless is_list(value) do
-          Mix.raise "Application description (:description) is not a character list, got: #{inspect value}"
+          Mix.raise(
+            "Application description (:description) is not a character list, got: #{
+              inspect(value)
```

Comment:
And I'm not sure about such splits of long lines with variables. How best to do in such cases?

Reply:
```
msg = "Application description (:description) is not a character list, got: #{inspect(value)}"
```

or if it still doesn't fit, break using `<>`:

```elixir
"Application description (:description) is not a character list, " <>
  "got: #{inspect(value)}"
```